akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1368,6 +1368,32 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
   def mapError(pf: PartialFunction[Throwable, Throwable]): javadsl.Flow[In, Out, Mat] =
     new Flow(delegate.mapError(pf))
 
+  /**
+   * Java API
```

Comment:
BTW, this is already in a Java only package, so no need for this

Reply:
good point. will remove.